### PR TITLE
Lit6620 der hom kah

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**

### CODE REVIEW

File: new/LIT6620DerHomKah.xml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                <listRelation>
+                    <relation name="saws:isDifferentTo" active="LIT6620DerHomKah" passive="LIT6614FourCreat"/>
+                    <relation name="saws:isDifferentTo" active="LIT6620DerHomKah" passive="LIT4958Priests"/>
+                    <relation name="saws:isDifferentTo" active="LIT6620DerHomKah" passive="LIT6469HomilyKahenat"/>
+                    <relation name="ecrm:P129_is_about" active="LIT6620DerHomKah" passive="PRS12519TwentyFour"/>
+                </listRelation>
+            </div><!---->
+
+
+                      <div type="bibliography"/>
```
Comment (eu-genia) [inline new/LIT6620DerHomKah.xml:64]: please remove the `div type="bibliography" `around the `div type="edition"`. thank you

File: new/LIT6620DerHomKah.xml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6620DerHomKah" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ድርሳን፡ ዘ፳ወ፬፡ ካህናተ፡ ሰማይ፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Dǝrsān za20wa4 kāhǝnāta samāy</title>
+                <title xml:lang="en" xml:id="t2">Homily for the commemoration of the Twenty-Four Elders of Heaven</title>
+                <editor role="generalEditor" key="AB"/>
```
Comment (eu-genia) [inline new/LIT6620DerHomKah.xml:11]: maybe add as a title variant "... heavenly priests" as they are often referred to that way

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Comment by MarcinKrawczuk** [comment]
Done (I hope)

wt., 10 maj 2022 o 04:13 Eugenia ***@***.***> napisał(a):

**Review by eu-genia — APPROVED** [review]

**Comment by eu-genia** [comment]
> Done (I hope) 
I corrected the nesting, you deleted too much and in the end still had the edition inside of bibliography ;)

**Pull request merged**